test/cypress/e2e/contributors.cy.js

```diff
@@ -0,0 +1,20 @@
+describe('Contributors', () => {
+  let allContributorsData;
+
+  before(() => {
+    cy.request('contributors.json').then(response => {
+      expect(response.status).to.eq(200);
+      allContributorsData = response.body;
+    });
+  });
+
+  context('when Hugo is started', () => {
```

Comment:
I do not understand the value of this context

Reply:
Seen in sync call, thanks.